packages/backend-function/package.json

```diff
@@ -18,6 +18,7 @@
   },
   "license": "Apache-2.0",
   "dependencies": {
+    "@aws-amplify/backend-output-schemas": "^0.6.0",
```

Comment:
I was wondering this too but since the new update is in this same PR, I'm thinking this and other versions will be bumped up to `beta` in the next release containing this PR.

Reply:
@edwardfoyle can you confirm this?

Reply:
Yeah changesets will do the version bumping / naming. This should be the version that currently exists in the backend-output-schemas package.json file